tools/tesseratos/assets/main_copy.pal.meta

```diff
@@ -0,0 +1,3 @@
+{
+    "id": "6f42ae5a-59d1-5df3-1720-83b8df6dd536"
```

Comment:
I would prefer if you generated a different UUID for this palette. Its very hard to spot the different between the two UUIDs.